problemsets-mat223/modules/module11-exercises.tex

```diff
@@ -228,5 +228,42 @@
 					A)=2$ $\Nullity(T) = \Nullity(A)=1$
 			\end{enumerate}
 		\end{solution}
+
+		\prob Give an example of a $3\times4$ matrix $M$ with the specified rank, or explain why one cannot exist.
+		\begin{enumerate}
+			\item $\Rank(M) = 0$
+			\item $\Rank(M) = 1$
+			\item $\Rank(M) = 3$
+			\item $\Rank(M) = 4$
+		\end{enumerate}
+
+		\begin{solution}
+			\begin{enumerate}
+				\item $\mat{0&0&0&0\\0&0&0&0\\0&0&0&0}$
+				\item $\mat{1&0&0&0\\0&0&0&0\\0&0&0&0}$
+				\item $\mat{1&0&0&0\\0&1&0&0\\0&0&1&0}$
+				\item A $3\times4$ matrix with rank 4 cannot exist. Two examples of justifications are:
+				1) Thinking about the induced transformation, the 3 dimensional domain cannot produce a 4 dimensional range (Rank-Nullity Theorem).
```

Comment:
Be more precise with your language here. Maybe spell out the argument. Every vector in the domain is a LC of e_1, e_2, e_3, so every vector in the range must be a LC of Ae_1, .... Thus the range cannot be more than three dimensional.